### DISCUSSION TITLE
Where do I find the log files?

### DISCUSSION THREAD

**Discussion by wulfsystems** [emersion/maddy#398] [Q&A] [answered]
Maybe bad question, but are there log files like for postfix "mail.log"?

**Comment by foxcpp** [comment, marked as answer]
Log output is written to stderr by default. If you run maddy via systemd unit the log is written to the system journal and can be viewed using `journalctl` (e.g. `journalctl -u maddy`). If you run it via some other method and stderr is not recorded - well, you have no log. You can add `log` directive to the config to enable logging to syslog or to a file.

e.g. 
```
$(local_domain) = ...

log syslog # send log to system's syslog daemon
log /var/log/maddy/mail.log # write to a file directly
```
